modules/mri_violations/test/mri_violationsTest.php

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
I don't think this blank line is supposed to be here. It's not much of an issue for tests, but blank lines can confuse/break PHP